extension/js/common/browser/browser-msg.ts

```diff
@@ -321,9 +330,30 @@ export class BrowserMsg {
     });
   }
 
+  public static alarmListen() {
+    const alarmListener = (alarm: { name: string }) => {
+      const alarmName = alarm.name;
+      const actionName = alarmName.split('_interval')[0];
+      if (BrowserMsg.INTERVAL_HANDLERS[actionName]) {
+        Catch.try(BrowserMsg.INTERVAL_HANDLERS[actionName])();
+      }
+      if (alarmName.includes('interval')) {
+        const splitAry = alarmName.split('_');
+        const ms = splitAry[splitAry.length - 1];
+        if (ms) {
+          // Recreate alarm so that interval can be called again
+          void chrome.alarms.create(alarmName, { when: Date.now() + parseInt(ms) });
```

Comment:
it's possible to create interval alarm using `periodInMinutes` property (https://developer.chrome.com/docs/extensions/reference/api/alarms#properties_1), what about using it instead of manually creating next alarm?

Reply:
Do we use intervals less than 30 seconds? It seems `createIntervalAlarm` is used only for `delete_expired` and it runs each minute, should be ok to use `periodInMinutes` there

Reply:
Got it